packages/frontend/src/hooks/useQueryError.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+import { useQueryClient } from 'react-query';
+import { useApp } from '../providers/AppProvider';
+
+const useQueryError = () => {
+    const queryClient = useQueryClient();
+    const [errorResponse, setErrorResponse] = useState<any>(null);
```

Comment:
We could type it with `ApiError` and remove the ts-ignore in line 14

Reply:
done